@gibman thanks for the PR, looking forward to having it in a release since it's quite annoying to keep powercycling my KLF 200 :)

I'm not too familiar with the automated checks involved here, but clicking through the details links brought me to this: https://dev.azure.com/home-assistant/Home%20Assistant/_build/results?buildId=8149&view=logs&jobId=2fa2c639-d05e-530d-034e-4fb413fcea4e&j=4e214fed-f310-53fa-0018-b38e7beb405b -- if you expand the "Run flake8" section you get a bunch of lint errors:

homeassistant/components/velux/__init__.py:27:38: W291 trailing whitespace
homeassistant/components/velux/__init__.py:50:26: W291 trailing whitespace
homeassistant/components/velux/__init__.py:52:1: W293 blank line contains whitespace
homeassistant/components/velux/__init__.py:54:1: D102 Missing docstring in public method
homeassistant/components/velux/__init__.py:54:5: E303 too many blank lines (2)
homeassistant/components/velux/__init__.py:59:1: W293 blank line contains whitespace
homeassistant/components/velux/__init__.py:60:81: W291 trailing whitespace
homeassistant/components/velux/__init__.py:66:5: E303 too many blank lines (2)
homeassistant/components/velux/__init__.py:68:49: W291 trailing whitespace
homeassistant/components/velux/__init__.py:71:1: W391 blank line at end of file
homeassistant/components/velux/__init__.py:71:1: W293 blank line contains whitespace

This might also be useful for checking locally so that you don't have to wait for the CI robots to point out your errors: https://developers.home-assistant.io/docs/en/development_testing.html#preventing-linter-errors

--- a/homeassistant/components/velux/__init__.py
+++ b/homeassistant/components/velux/__init__.py
@@ -6,7 +6,7 @@ from pyvlx import PyVLXException
 
 from homeassistant.helpers import discovery
 import homeassistant.helpers.config_validation as cv
-from homeassistant.const import (CONF_HOST, CONF_PASSWORD, EVENT_HOMEASSISTANT_STOP)
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, EVENT_HOMEASSISTANT_STOP
 
 DOMAIN = "velux"
 DATA_VELUX = "data_velux"
@@ -50,7 +50,6 @@ class VeluxModule:
         self._hass = hass
         self._domain_config = domain_config
 
-
     def setup(self):
         async def on_hass_stop(event):
             """Close connection when hass stops."""
@@ -62,10 +61,8 @@ class VeluxModule:
         password = self._domain_config.get(CONF_PASSWORD)
         self.pyvlx = PyVLX(host=host, password=password)
 
-
     async def async_start(self):
         """Start velux component."""
         _LOGGER.debug("Velux interface started")
         await self.pyvlx.load_scenes()
         await self.pyvlx.load_nodes()
-

--- a/homeassistant/components/velux/__init__.py
+++ b/homeassistant/components/velux/__init__.py
@@ -24,7 +24,7 @@ CONFIG_SCHEMA = vol.Schema(
 
 
 async def async_setup(hass, config):
-    """Set up the velux component."""    
+    """Set up the velux component."""
     try:
         hass.data[DATA_VELUX] = VeluxModule(hass, config[DOMAIN])
         hass.data[DATA_VELUX].setup()
@@ -47,17 +47,17 @@ class VeluxModule:
     def __init__(self, hass, domain_config):
         """Initialize for velux component."""
         self.pyvlx = None
-        self._hass = hass        
+        self._hass = hass
         self._domain_config = domain_config
-        
+
 
     def setup(self):
         async def on_hass_stop(event):
             """Close connection when hass stops."""
             _LOGGER.debug("Velux interface terminated")
             await self.pyvlx.disconnect()
-            
-        self._hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, on_hass_stop)             
+
+        self._hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, on_hass_stop)
         host = self._domain_config.get(CONF_HOST)
         password = self._domain_config.get(CONF_PASSWORD)
         self.pyvlx = PyVLX(host=host, password=password)
@@ -65,7 +65,7 @@ class VeluxModule:
 
     async def async_start(self):
         """Start velux component."""
-        _LOGGER.debug("Velux interface started")        
+        _LOGGER.debug("Velux interface started")
         await self.pyvlx.load_scenes()
         await self.pyvlx.load_nodes()
-        
+

--- a/homeassistant/components/velux/__init__.py
+++ b/homeassistant/components/velux/__init__.py
@@ -51,6 +51,7 @@ class VeluxModule:
         self._domain_config = domain_config
 
     def setup(self):
+        """Velux component setup"""
         async def on_hass_stop(event):
             """Close connection when hass stops."""
             _LOGGER.debug("Velux interface terminated")

